include/SDL3/SDL_audio.h

```diff
@@ -852,9 +852,12 @@ extern DECLSPEC void SDLCALL SDL_DestroyAudioStream(SDL_AudioStream *stream);
  * \param volume ranges from 0 - 128, and should be set to SDL_MIX_MAXVOLUME
  *               for full audio volume
  *
+ * \returns 0 on success or a negative error code on failure; call
```

Comment:
I think the pattern has been that there isn't a blank line between the parameters and the return value. Can you double check this?

Reply:
updated